### ISSUE TITLE
Missing required option: '--source <source>'

### ISSUE THREAD

**Issue by offSwitch** [leonchen83/redis-rdb-cli#31]
**Ask a question**
Feel free to ask any question about this tool. no need to fill following table.  

**Describe the bug**
![image](https://user-images.githubusercontent.com/11695496/156547480-0b5a2dc5-14a6-4490-9475-a53561a605d7.png)
![image](https://user-images.githubusercontent.com/11695496/156547598-63dcdf49-cb6e-4344-96d2-162198de690f.png)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version(run `rct --version` or `rct -V` and paste the information):**  
v0.8.2
**Additional context**

**Comment by leonchen83** [comment]
@offSwitch 
file suffix must be `.rdb` not `.db`
if use `--source`. do not add `=` character after `--source` option

**Comment by offSwitch** [comment]
I think it is unreasonable to require the file name suffix to be rdb, because the file name is customizable

**Comment by leonchen83** [comment]
yes. it as a precheck before truely parse`rdb`.  I already add `bug` label on this issue . If I have time I will fix it.
